### DISCUSSION TITLE
Simple_table to docx generation not performed as expected.

### DISCUSSION THREAD

**Discussion by mmienkina** [jgm/pandoc#9737] [Q&A]
Hello,
Using pandoc-3.1.13 and generating table to docx.

Table representation in markdown (close to simple_tables extension generated by tabulate with tablefmt="plain") :
```
  Start addr [hex]  End addr [hex]      Size [KB]   Description
------------------  ----------------  ------------  -------------
          46000000  467FFFFF                  8192  PBRIDGE
          46800000  46FFFFFF                  8192  Reserved
          47000000  477FFFFF                  8192  Reserved
          47800000  47FFFFFF                  8192  Reserved
```

Generated docx:
![image](https://github.com/jgm/pandoc/assets/169362001/9ae3eaa6-1385-488f-81b4-2e469897bc02)

Grid extension table works fine but I prefer simple_tables because width of simple_tables is nicely scaled to table content vs. grid one looking like scaled to the window:
![image](https://github.com/jgm/pandoc/assets/169362001/8e0779d8-3a1f-4df2-8928-b4ec4b27d5e2)

Workaround is to generate table using grid extension and then manually scale table using (Layout->Autofit to Content) that requires manual interaction with generated file and hence not preferred.

Can you please suggest on how to get correct results from simple_tables extension?

Thanks,
Martin M.

**Comment by jgm** [comment]
Here's what it looks like when I generate a docx from that table using current pandoc:
<img width="416" alt="image" src="https://github.com/jgm/pandoc/assets/3044/f7ffea1c-9dce-4d4c-b50d-9656bb3f6909">

That looks fine.  Do you get different results if you use the latest pandoc version?
(Note: this is Word for Mac.  What are you using to open the docx?)

**Reply by mmienkina** [reply]
Hello John,
Thank you very much for your answer.
Having confirmation it works, I have look at other places and found issue in my script embedding md tables in a single md file. This issue caused wrong formatting of table headers (see below) in a big md file used by pandoc.exe to generate docx.
```
Start addr [hex]  End addr [hex]      Size [KB]   Description
------------------  ----------------  ------------  -------------
```
After fixing issue and providing correct headers my tables in docx looks as expected.
Thank you and kind regards,
Martin M.
